### ISSUE TITLE
THis is wrong

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#601]

**Comment by ThePokerDude** [comment]
the correct bidding for south is pass intead of 1s.
It seems there is no hardcoded handling of take out double and responds to it implemented.
I think this should be done as it is a very common situation.

A pretty good description can be found:
https://www.bridgebum.com/takeout_double.php

The rule is - after take out double and lowest level respond, the player who doubled answers:
pass = 11-15/16 HCP
raise = 17-19 HCP
new suit 19+ (according to bridgebum). I play it as 17+ HCP 5 card suit

**Comment by ThorvaldAagaard** [comment]
Yes, and there was an interesting comment on Bridgewinners about T/O.

After a T/O double on a higher level, new suit from doubler is just showing the other suits. I did not find any documentation on that. It could also be interesting with an implementation of Equal Level Conversion.

**Comment by ThePokerDude** [comment]
Fixed in 8635
![image](https://github.com/user-attachments/assets/5869bf34-73c9-43ce-a854-60c0e2a29d2e)
